stdlib/asm/crypto/stark/random_coin.masm

```diff
@@ -174,33 +168,54 @@ export.init_seed
 
     ## main segment width is 71 and there are 1 auxiliary segments
     ## of width 7 using 16 random extension field elements
-    push.0x47010710 
+    push.0x47010710 swap
     ## field modulus bytes (2 field elements)
     push.0x01 # lower half of the modulus
     push.0xffffffff # upper half of the modulus
     ## field extension and FRI parameters
     ## field extension degree || FRI folding factor || FRI remainder polynomial max degree || blowup factor
```

Comment:
This comment still mentions the blowup

Reply:
Yeah, but this reference is with respect to the "hard-coded" value inside the field element encoding the FRI protocol parameters.